### PR TITLE
#788 fixed bug regards that a new users haven't their avatars 

### PR DESCRIPTION
**Pull request by dmzaytsev**
For #788 
* modified `RqAlias` to update user's photo from the request
* added a unit test

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev Thanks, someone will review your pull request soon

**Comment by karato** [comment]
@pinaf do this review

File: netbout-web/src/main/java/com/netbout/rest/RqAlias.java [hunk 1]
```diff
@@ -102,6 +102,7 @@ public User user() throws IOException {
      * Get alias.
      * @return Alias
      * @throws IOException If fails
+     * @todo
```
Comment (pinaf) [inline netbout-web/src/main/java/com/netbout/rest/RqAlias.java]: @dmzaytsev this doesn't look right

**Comment by pinaf** [comment]
@dmzaytsev 1 comment

File: netbout-web/src/main/java/com/netbout/rest/RqAlias.java [hunk 2]
```diff
@@ -111,7 +112,16 @@ public Alias alias() throws IOException {
                 "/login/start"
             );
         }
-        return aliases.iterate().iterator().next();
+        final Alias alias = aliases.iterate().iterator().next();
+        if (alias.photo().equals(Alias.BLANK)) {
```
Comment (yegor256) [inline netbout-web/src/main/java/com/netbout/rest/RqAlias.java]: @dmzaytsev also, keep in mind, that in this design we will set user avatar only once. Later, when the user changes the picture in Github, we won't catch that change. We will continue to use the old picture. I think that the only solution is to add randomization here (pseudo code):

```
if (alias.photo().equals(Alias.BLANK) || new Random().nextInt(10) == 5) {
  // ...
}
```

Comment (yegor256) [inline netbout-web/src/main/java/com/netbout/rest/RqAlias.java]: @dmzaytsev wait... maybe we can do it even better. First, read current `photo()` and then compare it with the avatar from `identity`. If they are not equal, save the new value.

**Comment by dmzaytsev** [comment]
@pinaf @yegor256 done
please take a look

**Comment by yegor256** [comment]
@dmzaytsev looks good to me

**Comment by pinaf** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy pls

**Comment by elenavolokhova** [comment]
@pinaf 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
Only one issue was found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by pinaf** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here

**Comment by karato** [comment]
> @karato Quality is acceptable here

@elenavolokhova thanks for the review, we'll try better next time

**Comment by karato** [comment]
@pinaf I added 10 mins to @elenavolokhova (for QA review) in transaction 69441612; Thank you, I have added **19 mins** to your account in payment/transaction "`AP-5X97578017643452F`", time consumed: 29 hours and 50 mins; you're getting extra minutes here (c=4); +19 added to your rating, at the moment it is: [+3903](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**